pyscf/adc/uadc_amplitudes.py

```diff
@@ -742,6 +742,96 @@ def compute_energy(myadc, t1, t2, eris):
+    OPDM_a[nocc_a:, nocc_a:] += lib.einsum('ijAa,ijCa->AC', t1_ccee_abab, t1_ccee_abab, optimize = einsum_type)
+
+    #OPDM_a[:nocc_a, nocc_a:]  = lib.einsum('IC->IC', t1_ce_aa, optimize = einsum_type).copy()
+    OPDM_a[:nocc_a, nocc_a:] += lib.einsum('IC->IC', t2_ce_aa, optimize = einsum_type).copy()
+    
+    #OPDM_a[:nocc_a, nocc_a:] += 1/2 * lib.einsum('IiCa,ia->IC', t1_ccee_aaaa, t1_ce_aa, optimize = einsum_type)
+    #OPDM_a[:nocc_a, nocc_a:] += 1/2 * lib.einsum('IiCa,ia->IC', t1_ccee_abab, t1_ce_bb, optimize = einsum_type)
+
+    #OPDM_a[nocc_a:, :nocc_a]  = lib.einsum('LA->AL', t1_ce_aa, optimize = einsum_type).copy()
+    OPDM_a[nocc_a:, :nocc_a]  += lib.einsum('LA->AL', t2_ce_aa, optimize = einsum_type).copy()
+    #OPDM_a[nocc_a:, :nocc_a]  += 1/2 * lib.einsum('LiAa,ia->AL', t1_ccee_aaaa, t1_ce_aa, optimize = einsum_type)
+    #OPDM_a[nocc_a:, :nocc_a]  += 1/2 * lib.einsum('LiAa,ia->AL', t1_ccee_abab, t1_ce_bb, optimize = einsum_type)
+    #---
+    OPDM_b[:nocc_b, :nocc_b]  = lib.einsum('il->il', np.identity(nocc_b), optimize = einsum_type).copy()
+    OPDM_b[:nocc_b, :nocc_b] -= 1/2 * lib.einsum('ijab,ljab->il', t1_ccee_bbbb, t1_ccee_bbbb, optimize = einsum_type)
+    #OPDM_b[:nocc_b, :nocc_b] -= lib.einsum('ia,la->il', t1_ce_bb, t1_ce_bb, optimize = einsum_type)
+    OPDM_b[:nocc_b, :nocc_b] -= lib.einsum('jiab,jlab->il', t1_ccee_abab, t1_ccee_abab, optimize = einsum_type)
+
+    #OPDM_b[nocc_b:, nocc_b:] = lib.einsum('ia,ic->ac', t1_ce_bb, t1_ce_bb, optimize = einsum_type)
+    OPDM_b[nocc_b:, nocc_b:] += lib.einsum('ijba,ijbc->ac', t1_ccee_abab, t1_ccee_abab, optimize = einsum_type)
+    OPDM_b[nocc_b:, nocc_b:] += 1/2 * lib.einsum('ijab,ijcb->ac', t1_ccee_bbbb, t1_ccee_bbbb, optimize = einsum_type)
+
+    #OPDM_b[:nocc_b, nocc_b:]  = lib.einsum('ic->ic', t1_ce_bb, optimize = einsum_type).copy()
+    OPDM_b[:nocc_b, nocc_b:]  += lib.einsum('ic->ic', t2_ce_bb, optimize = einsum_type).copy()
+    #OPDM_b[:nocc_b, nocc_b:]  += 1/2 * lib.einsum('ijca,ja->ic', t1_ccee_bbbb, t1_ce_bb, optimize = einsum_type)
+    #OPDM_b[:nocc_b, nocc_b:]  += 1/2 * lib.einsum('jiac,ja->ic', t1_ccee_abab, t1_ce_aa, optimize = einsum_type)
+
+    #OPDM_b[nocc_b:, :nocc_b]   = lib.einsum('la->al', t1_ce_bb, optimize = einsum_type).copy()
+    OPDM_b[nocc_b:, :nocc_b]  += lib.einsum('la->al', t2_ce_bb, optimize = einsum_type).copy()
+    #PDM_b[nocc_b:, :nocc_b]  += 1/2 * lib.einsum('liab,ib->al', t1_ccee_bbbb, t1_ce_bb, optimize = einsum_type)
+    #PDM_b[nocc_b:, :nocc_b]  += 1/2 * lib.einsum('ilba,ib->al', t1_ccee_abab, t1_ce_aa, optimize = einsum_type)
+
+    #---
+    # print("OPDM_a_H " + str(np.linalg.norm(OPDM_a - OPDM_a.transpose(1,0))))
+    # print("OPDM_b_H " + str(np.linalg.norm(OPDM_b - OPDM_b.transpose(1,0))))
+
+    # print("OPDM_a_trace " + str(np.einsum('pp',OPDM_a)))
+    # print("OPDM_b_trace " + str(np.einsum('pp',OPDM_b)))
+    # exit()
+
```

Comment:
please remove the testing code, or use `lib.logger` to output the debugging info